### ISSUE TITLE
Change Primary Colour used for links

### ISSUE THREAD

**Issue by cooper667** [gliff-ai/document#34]
Our default light green looks good on a dark background, but when used for text on a light background, it doesn't have enough contrast and is hard to read/not great for accessibility

![image](https://user-images.githubusercontent.com/3502798/152508742-59b7266a-5c43-4442-9721-b33045c86c6e.png)
